HomeWork_03/DZ3.py

```diff
@@ -52,28 +49,8 @@
 
 #insert (int) to  (str)#
 # Zadanie 6
-a=str(2)
-b=str(5)
-c=str(6)
-d=(a+b+c)
+a= 2
+b= 5
+c= 6
+d=str(a+b+c)
```

Comment:
`d` should be a string created by concatenation of `a, b, c` as string. Like this: `d = str(a) + str(b) + str(c)`